src/test/Makefile

```diff
@@ -17,7 +17,7 @@ ROOT = ../..
 OBJECT_DIR = ../../obj/test
 TARGET_DIR = $(USER_DIR)/target
 
-include $(ROOT)/make/system-id.mk
+include $(ROOT)/mk/system-id.mk
```

Comment:
Maybe change to MAKE_SCRIPT_DIR here too?

Reply:
Updated.